### PR TITLE
SPCTR-2799: Improvement: Removal of extra variable for props

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Removal of extra varialble for props by replacing with spread operator.

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/taxonomy-list/render.js [hunk 1]
```diff
@@ -11,7 +11,6 @@ const Render = ( props ) => {
 		};
 	}, [] );
 	const { categoriesList } = props;
-	props = props.parentProps;
 	// Caching all Props.
 	const { attributes, deviceType } = props;
```
[the remark this answers is not in the document]
Comment (rajmohanreddy774) [reply src/blocks/taxonomy-list/render.js]: Made changes

File: src/blocks/taxonomy-list/settings.js [hunk 2]
```diff
@@ -26,7 +26,6 @@ import UAGTextControl from '@Components/text-control';
 import apiFetch from '@wordpress/api-fetch';
 const Settings = ( props ) => {
 	const { taxonomyList, termsList } = props;
-	props = props.parentProps;
 
 	// Caching all Props.
 	const { attributes, setAttributes, deviceType } = props;
```
[the remark this answers is not in the document]
Comment (rajmohanreddy774) [reply src/blocks/taxonomy-list/settings.js]: Made changes

File: src/blocks/testimonial/settings.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ import UAGAdvancedPanelBody from '@Components/advanced-panel-body';
 import { memo } from '@wordpress/element';
 
 const Settings = ( props ) => {
-	props = props.parentProps;
+
 	const blockName = props.name.replace( 'uagb/', '' );
 	const { setAttributes, attributes, deviceType } = props;
```
[the remark this answers is not in the document]
Comment (rajmohanreddy774) [reply src/blocks/testimonial/settings.js]: done

File: src/blocks/timeline/post-timeline/render.js [hunk 4]
```diff
@@ -24,7 +24,7 @@ const Render = ( props ) => {
 		};
 	}, [] );
 	const { latestPosts } = props;
-	props = props.parentProps;
+	
 	const blockName = props.name.replace( 'uagb/', '' );
 	const { attributes, className, setAttributes, deviceType } = props;
```
[the remark this answers is not in the document]
Comment (rajmohanreddy774) [reply src/blocks/timeline/post-timeline/render.js]: done changes

**Review by rahulvarma722 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**